specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -1773,6 +1829,104 @@
         }
       }
     },
+    "AcsCallParticipantBaseEventData": {
+      "description": "Schema of common properties of participant events",
+      "type": "object",
+      "properties": {
+        "user": {
+          "description": "The communication identifier of the participant",
+          "$ref": "../../../../../communication/data-plane/Common/stable/2023-11-15/common.json#/definitions/CommunicationIdentifierModel"
+        },
+        "displayName": {
+          "description": "The display name of the participant.",
+          "type": "string"
+        },
+        "participantId": {
+          "description": "The Id of the participant.",
+          "type": "string"
+        },
+        "userAgent": {
+          "description": "The user agent of the participant. ",
+          "type": "string"
+        }
+      }
+    },
+    "AcsCallingEventBaseProperties": {
+      "description": "Schema of common properties of all calling events",
+      "type": "object",
+      "properties": {
+        "startedBy": {
+          "description": "The communication identifier of the call started by",
+          "$ref": "../../../../../communication/data-plane/Common/stable/2023-11-15/common.json#/definitions/CommunicationIdentifierModel"
+        },
+        "serverCallId": {
+          "description": "The callId of the server",
+          "type": "string"
+        },
+        "group": {
+          "description": "The group metadata",
+          "$ref": "#/definitions/AcsCallingEventGroupProperties"
+        },
+        "isTwoParty": {
+          "description": "The isTwoParty metadata",
+          "type": "boolean"
+        },
+        "correlationId": {
```

Comment:
you are missing the property room in the callingEventsBase. 
![{24C69891-87EB-42CC-AFD6-F4211B51CB7E}](https://github.com/user-attachments/assets/8a83b241-3f36-4378-88cc-22c842871f8a)